core/src/main/java/org/openstack4j/openstack/telemetry/domain/CeilometerStatistics.java

```diff
@@ -34,17 +46,22 @@
 
 	private Integer period;
 
-	@JsonProperty("period_start")
+   @JsonProperty("period_start")
+   private String periodStartStr;
+
+   @JsonProperty("period_end")
+   private String periodEndStr;
+
 	private Date periodStart;
 
-	@JsonProperty("period_end")
 	private Date periodEnd;
 
 	private Double sum;
 
 	private String unit;
 
-	private String groupby;
+   @JsonProperty("groupby")
+   private Map<String, Object> groupBy;
```

Comment:
It actually didn't work with the second type as a String. It was just ignored by Jackson. This field is actually a bit confusing, it looks like:
 ```
{"project_id":"some_id"}
```
It itsn't necessary for this PR so I think I will revert the change.